pkg/controller/pod_volume_backup_controller.go

```diff
@@ -783,13 +788,25 @@ func (r *PodVolumeBackupReconciler) setupExposeParam(pvb *velerov1api.PodVolumeB
 		}
 	}
 
+	hostingPodTolerations := []corev1api.Toleration{}
```

Comment:
Consider implementing a cache for tolerations keyed by backup name to reduce API calls. Currently, every pod creation triggers multiple API calls to fetch DaemonSet information. For large-scale operations (e.g., backing up 100 PVCs), this could generate 500+ API calls. A simple in-memory cache with TTL would significantly improve performance.